### ISSUE TITLE
Timed out waiting for the lifecycle-server to start requires a reset every time

### ISSUE THREAD

**Issue by F21** [docker/for-win#5069]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 19BA82D8-4BB7-4F2D-9ACB-2DD62C7BB447/20191105215650

### Expected behavior
Docker for desktop should start correctly
### Actual behavior
Docker for desktop fails to start with the following error:
````
Timed out waiting for the lifecycle-server to start.
This may be the result of a slow environment or there may be a real problem.
This timeout can be increased by editing settings.json, value 'lifecycleTimeoutSeconds'.
This file is usually in C:\Users\(your username)\AppData\Roaming\Docker

   at Docker.Core.Pipe.NamedPipeClient.<TrySendAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Docker.Core.Pipe.NamedPipeClient.Send(String action, Object[] parameters)
   at Docker.Actions.DoStart(SynchronizationContext syncCtx, Boolean showWelcomeWindow, Boolean executeAfterStartCleanup)
   at Docker.Actions.<>c__DisplayClass25_0.<Start>b__0()
   at Docker.ApiServices.TaskQueuing.TaskQueue.<>c__DisplayClass18_0.<.ctor>b__1()
````

The only way to get it to start correctly is to "Reset to Factory Defaults". However, having to do this every time you need to start docker means you lose all your images and data, which is quite annoying.

In the Hyper-V Manager, the DockerDesktopVM is up and running correctly, so it may be that there's an issue detecting the VM as running correctly.
### Information

  - Windows Version: Windows 10 1903 64-bit 18362.449
  - Docker Desktop Version: 2.1.5.0 and 2.1.4.0 (Edge)
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No
  - The current solution is to down grade to 2.1.3.0 (Edge)

### Steps to reproduce the behavior
  1. Install Docker Desktop for Windows 2.1.5.0 or 2.1.4.0
  2. Start docker and use it to do some work.
  3. Shut down docker desktop and restart it.
  4. Docker Desktop fails to start and the only way to fix it is to reset to factory defaults.
  5. Use docker to do some work.
  6. Shut down docker desktop and restart it.
  7. Docker Desktop fails to start and the only way to fix it is to reset to factory defaults.

**Comment by yudhir** [comment]
I got the issue when storage was running low and the build process actually stopped . Restart of the computer solved that . It is functioning correctly now.

**Comment by F21** [comment]
Seems to be fixed in 2.1.6.0
